non_logged_in_area/templates/base_non_logged_in.html

```diff
@@ -11,6 +11,12 @@
 
     <title>Volunteer Planner</title>
 
+    <link rel="icon" type="image/x-icon" href="{% static "img/favicon.ico" %}">
```

Comment:
I think those links need to go to `<ROOT>\templates/base.html`, too.

Reply:
Arent't they?

Reply:
Yes, they are. I didn't see it, sorry. 🙈

Reply:
Yes, they are. I didn't see it, sorry. 🙈